CesiumGS/cesium
# Add japan building data sandcastle

# Description

- create a sandcastle to highlight our new Japan building dataset
- Includes a few POIs in different locations to showcase the data
    - I planned to add translations for the dropdown, those should come shortly.
- I opted for a light that comes from the camera to avoid nighttime making the buildings darker
- I turned on antialiasing as it helps a lot with the edges of buildings especially in the areas that have actual imagery textures. @ggetz is there anything else that might make sense as "easy wins" to increase the visual quality?







## Issue number and link

No issue





## Testing plan

- Run `npm start`
- open sandcastle and navigate to the [Japan Buildings one](http://localhost:8080/Apps/Sandcastle/index.html?src=Japan%20Buildings.html)
- Test the sandcastle and make sure it behaves well
- Make sure it shows up in search and the categories make sense
- Make sure the thumbnail looks good



# Author checklist
